src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5695,10 +5695,7 @@
   <data name="ERR_InvalidExperimentalDiagID" xml:space="preserve">
     <value>The diagnosticId argument to the 'Experimental' attribute must be a valid identifier</value>
   </data>
-  <data name="WRN_LockTypeUnsupported" xml:space="preserve">
-    <value>A value of type 'System.Threading.Lock' in SyncLock will use likely unintended monitor-based locking. Consider manually calling 'Enter' and 'Exit' methods in a Try/Finally block instead.</value>
-  </data>
-  <data name="WRN_LockTypeUnsupported_Title" xml:space="preserve">
+  <data name="ERR_LockTypeUnsupported" xml:space="preserve">
     <value>A value of type 'System.Threading.Lock' in SyncLock will use likely unintended monitor-based locking. Consider manually calling 'Enter' and 'Exit' methods in a Try/Finally block instead.</value>
```

Comment:
This error should be reworded, since it should no longer imply that `Lock` is supported, but a warning.

Reply:
Would something like this work? "A value of type 'System.Threading.Lock' is not supported in SyncLock. Consider manually calling 'Enter' and 'Exit' methods in a Try/Finally block instead."

Reply:
> This error should be reworded, since it should no longer imply that Lock is supported, but a warning.

What wording do you suggest?

Reply:
Something like what @jjonescz suggested would work.